client/components/header-cake/index.jsx

```diff
@@ -40,16 +39,16 @@ module.exports = React.createClass( {
 		return (
 			<Card className={ classes }>
 				<div className="header-cake__corner">
-					<a className="header-cake__back" onClick={ this.props.onClick }>
-						<Gridicon icon="chevron-left" size={ 16 } />
-						<span className="header-cake__back-text">{ this.props.backText || this.translate( 'Back' ) }</span>
-					</a>
+					<HeaderCakeBack text={ backText } onClick={ this.props.onClick } />
 				</div>
-				<span className="header-cake__title" onClick={ this.props.onTitleClick }>
+				<div className="header-cake__title" onClick={ this.props.onTitleClick }>
 					{ this.props.children }
-				</span>
-				<div className="header-cake__corner" />
+				</div>
+				<div className="header-cake__corner is-spacer">
+					<HeaderCakeBack text={ backText } />
```

Comment:
What is this doing? It looks like it's invisible, but I never see it appear in the code.